src/analysis/retail/paladin/holy/modules/dragonflight/tier/Tier29FourSet.tsx

```diff
@@ -14,15 +14,24 @@ import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
 
 const CDR = 1000;
 
-// todo: refactor file to match tier set bonus in dragonflight
+// https://www.wowhead.com/news/paladin-season-1-dragonflight-tier-set-bonuses-guide-writer-first-impressions-329300
 
-class Tier28FourSet extends Analyzer {
+//https://www.wowhead.com/beta/spell=393672/paladin-holy-class-set-4pc
+// Paladin Holy Class Set 4pc
+// Requires Paladin
+// Holy Light, Flash of Light, Light of Dawn, Word of Glory, and Bestow Faith healing is increased by 6% and their critical effects
+// increase the damage or healing of your next Holy Shock by 20%.
+
+// TODO: convert this from Tier28FourSet Tier29FourSet
+// File has just currently been renamed
+
+class Tier29FourSet extends Analyzer {
```

Comment:
you are better off deleting the old tier analyzer and writing a new one from scratch than trying to repurpose it.

Reply:
noted!